src/main/java/com/githubai/bot/AppController.java

```diff
@@ -13,6 +14,11 @@ public String index() {
         return "ok";
     }
 
+    @GetMapping("/health")
+    public ResponseEntity<Map<String, String>> healthCheck() {
+        return ResponseEntity.ok(Map.of("status", "ok"));
```

Comment:
please add a TODO comment line above because basically we have a copy paste of the code from the postHandler here. We should unify both